client/src/frontend/components/composed/EntityCollection/patterns/JournalVolumes.js

```diff
@@ -0,0 +1,61 @@
+import React from "react";
+import PropTypes from "prop-types";
+import { ProjectCollectionIcon } from "../parts";
+import EntityCollection from "../EntityCollection";
+import Journal from "frontend/components/journal";
+
+function JournalVolumesEntityCollection({
+  journal,
+  volumes,
+  meta,
+  paginationProps,
+  ...passThroughProps
+}) {
+  if (!journal || !volumes) return null;
+
+  const { title } = journal.attributes ?? {};
+
+  return (
+    <EntityCollection
+      title={`${title}: Volumes`}
+      IconComponent={props => (
+        <ProjectCollectionIcon {...props} collection={journal} />
+      )}
```

Comment:
I didn't put a title link here because I couldn't think of anything that would make sense.